docs/client/full-api/api/templates.md

```diff
@@ -108,6 +141,15 @@ This group of callbacks is most useful for cleaning up or undoing any external
 effects of `created` or `rendered` groups. This group fires once and is the last
 callback to fire.
 
+```javascript
+Template.myPictures.onDestroyed(function () {
+  // deregister from some central store
+  var positionInStore = GalleryTemplates.indexOf(this);
```

Comment:
I would do this with `_.without`, http://underscorejs.org/#without

But maybe it's not good to use a semi-obscure function.